### PR TITLE
[CFM-1138] Non-Javascript fallback for Activity Stream

### PR DESCRIPTION
**Pull request by guytepper**
https://issuetracker.amplexor.com/jira/browse/CFM-1138

### CODE REVIEW

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/base/_global.scss [hunk 1]
```diff
@@ -83,3 +83,7 @@ div.contextual-links-wrapper {
   height: 100%;
   overflow-y: scroll;
 }
+
+.nojs-hidden {
+  display: none !important;
+}
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/base/_global.scss]: new line ...

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/plugins/content_types/activity_stream/activity-stream.tpl.php [hunk 2]
```diff
@@ -8,12 +8,12 @@
 <div class="activity-stream" ng-controller="ActivityCtrl">
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/plugins/content_types/activity_stream/activity-stream.tpl.php]: @guytepper 
Isn't it better to add the `nojs-hidden` on this element ?

Comment (guytepper) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/plugins/content_types/activity_stream/activity-stream.tpl.php]: Yup, thanks.

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.path_breadcrumbs.inc [hunk 3]
```diff
@@ -1,5 +1,4 @@
 <?php
-
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.path_breadcrumbs.inc]: @guytepper 
Revert this change.

**Comment by soniCaH** [comment]
@guytepper is this branch still relevant?

**Comment by ordavidil** [comment]
Closing in favor of #1456
